README.md

```diff
@@ -0,0 +1,35 @@
+Manifest-db
+=======
+
+Stores all the  manifests and corresponding image-infos to test OSbuild.
+
+### Update workflow
+
+The update workflow starts with a github action that is manually triggered. This
+GH action is named `Update DB`. It will firstly clone the latests
+`osbuild-composer` and copy all its manifests into the database. Secondly the
+action will create a commit and push this on the corresponding gitlab ci
+project
+(https://gitlab.com/redhat/services/products/image-builder/ci/manifest-db).
+Thirdly, after the push is completed, the action triggers a special pipeline on
+the CI. This pipeline builds every manifests from the DB. Once building is done,
```

Comment:
`manifests` -> `manifest`

Reply:
it should not be plural ?

Reply:
Nah, `every` is used before a singular noun, see https://dictionary.cambridge.org/dictionary/english/every